### PR TITLE
Ch ch changes

### PR DESCRIPTION
**Pull request by wooorm**
Hi! 👋

I updated dependencies, updated the code for the latest remark, added tests, linting (I think you’re using standard, right?).

I also rewrote some of the internals for the plugin, I \*think\* this is still correct and what you want this rule to do, could you check thoroughly though?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,57 +6,62 @@ This [remark-lint](https://github.com/wooorm/remark-lint) rule was created for [
...
 ## Using the rule
 
 ### Via `.remarkrc`
 
 ```bash
-npm install -g remark
-npm install -g remark-lint
-npm install remark-lint-appropriate-heading # local install!
+npm install -g remark-cli
+npm install remark-lint remark-lint-alphabetize-lists
```
Comment (RichardLitt) [inline README.md]: Should be remark-lint-appropriate-heading, no?

Comment (wooorm) [inline README.md]: Yes you’re right!

**Comment by wooorm** [comment]
Done!

**Comment by RichardLitt** [comment]
Thank you! Merged and updated. 🎉

**Comment by wooorm** [comment]
Perfect!

**Pull request merged**